packages/flutter/lib/src/material/popup_menu.dart

```diff
@@ -145,17 +207,111 @@ class _PopupMenuRoute<T> extends PopupRoute<T> {
   Color get barrierColor => null;
 
   Widget buildPage(BuildContext context, Animation<double> animation, Animation<double> forwardAnimation) {
-    return new _PopupMenu(route: this);
+    int selectedIndex = -1;
+    if (initialValue != null) {
+      for (int i = 0; i < items.length; i++)
+        if (initialValue == items[i].value) {
+          selectedIndex = i;
+          break;
+        }
+    }
+    final Size screenSize = MediaQuery.of(context).size;
+    return new SizedBox(
+      width: screenSize.width,
```

Comment:
I would expect this to just get its size by the normal layout protocol negotiating a "biggest" size from a constraint.